.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,13 +30,10 @@
   If an item is not applicable, you can add "N/A" to the end.
 -->
 
-- [ ] Code is up-to-date with the `master` branch
-- [ ] `npm run format` to apply prettier formatting
-- [ ] `npm run test` passes with this change
-- [ ] This pull request links relevant issues as `Fixes #0000`
-- [ ] There are new or updated unit tests validating the change
-- [ ] Documentation has been updated to reflect this change
-- [ ] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)
+-   [ ] Code is up-to-date with the `master` branch
+-   [ ] This pull request links relevant issues as `Fixes #00000`
+-   [ ] There are new or updated unit tests validating the change
+-   [ ] Documentation has been updated to reflect this change
```

Comment:
Could we re-add the link to CONTRIBUTING.md somewhere else in the template, to make sure contributors see it?